acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/commitboost/CommitBoostGetPubKeysAcceptanceTest.java

```diff
@@ -126,6 +132,41 @@ void listCommitBoostPublicKeys() {
     }
   }
 
+  @Test
+  void listEmptyCommitBoostPublicKeys() throws Exception {
```

Comment:
test name is a bit confusing to me. Maybe `deletingCommitBoostKeysRemovesPublicKeys` or something like that.